.github/workflows/ci.yml

```diff
@@ -5,13 +5,27 @@ on:
     branches:
       - "*"
       - "*/*"
+  pull_request:
+    branches:
+      - "*"
+      - "*/*"
 
 jobs:
   build:
     strategy:
       matrix:
-        os: [ubuntu-latest, macos-13, macos-latest, windows-latest]
-        version: [v2.0.0-alpha.130, v2.0.0-alpha.164, latest, ""]
+        os: [ubuntu-latest, macos-latest, windows-latest]
+        include:
+          - version: v2.0.0-alpha.130
+            is_legacy: true
+          - version: v2.0.0-alpha.164
+            is_legacy: true
+          - version: v2.0.0-alpha.189
```

Comment:
Can you leave the 189, latest, and "", out?
Ideally we should never need to touch the "include" part anymore.